### ISSUE TITLE
Create Aircraft File Validation

### ISSUE THREAD

**Issue by kmartino2168** [openscope/openscope#1263]
Related to #94.

It would be nice to have automated tests that validate the aircraft JSON files.  Right now, it's possible for an aircraft file to be invalid (e.g.: miscapitalized keys) and the for no one to ever realize it because some exceptions get swallowed.  This was the cause of #1259.

Checking the actual values of the file might be a bit impractical (that would require us to have a infallible reference sheet for the test to check against), but we could certainly at least make sure the file is **_valid_**.

**Comment by sam-irl** [comment]
I know it's for airports, but https://github.com/openscope/validator could be a starting point for this

**Comment by n8rzz** [comment]
Good call!  That would actually be pretty easy to implement. Well, maybe. Totally possible though.
